draft-ietf-jsonpath-base.md

```diff
@@ -2164,7 +2246,7 @@ The descendant operators, starting with `..`, borrowed from {{E4X}}, are similar
 The array slicing construct `[start:end:step]` is unique to JSONPath,
 inspired by {{SLICE}} from ECMASCRIPT 4.
 
-Filter expressions are supported via the syntax `?<boolean expr>` as in
+Filter expressions are supported via the syntax `?<expr>` as in
```

Comment:
This would need the same treatment as https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/403/files#r1113477827 if we decide to do that.

Reply:
I don't understand what you mean by "do that" (and the line has already changed into ?<logical-expr>)...

Reply:
"do that" == "follow Glyn's suggestion," which you did, and you changed it here, too.  This is resolved.